### DISCUSSION TITLE
How to model a substation with a backup generator

### DISCUSSION THREAD

**Discussion by nino-cmd** [e2nIEE/pandapower#1868] [Q&A]
Hello everyone....sorry to bug ya on this but just a quick question. I wanna simulate a huge grid where a substation has a transformer but also a diesel generator as a backup option. How do i present that in pandapower?? (on the same line (bus))?? or maybe present it somehow differently dont know

thank you in advance and really hoping to hear from you soon

**Comment by jkisse** [comment]
You can create a generator that is connected to the hv_bus or lv_bus of the substation's transformer
